### DISCUSSION TITLE
New Source: Open-Meteo

### DISCUSSION THREAD

**Discussion by ericpark** [airbytehq/airbyte#35310] [New Connector Request]
Hi, I would like to create a new connector for [Open-Meteo]( https://open-meteo.com/). Open-Meteo provides historical and forecast weather data without requiring authorization. Open-Meteo provides much more information compared to OpenWeather. 

Docs: [API Documentation](https://open-meteo.com/en/docs#)

I used this API when following an airbyte connector tutorial and would like to share it with the community. I believe I am pretty close to submitting a pull request (if this does not overlap with any existing works and complies with the product direction). 

Thanks!

**Comment by marcosmarxm** [comment]
Nice @ericpark are you using the low-code or connector builder to create the connector?

**Comment by ericpark** [comment]
@marcosmarxm , I used the python CDK because I needed to perform a zip on the response data. I tried the connector builder but it didn't seem possible to parse the data properly.

[a comment by an automated account is not shown]
